src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -95,6 +96,15 @@ const GWASContainer = () => {
         selectedCohort={state.selectedStudyPopulationCohort}
         outcome={state.outcome}
       />
+      <DismissibleMessagesList
```

Comment:
A drawback of this approach is that the error messages will not render near the input that causes the error. But that might be a reasonable trade off